torchvision/models/efficientnet.py

```diff
@@ -32,15 +32,23 @@
 
 
 class SqueezeExcitation(nn.Module):
-    def __init__(self, input_channels: int, squeeze_channels: int):
+    def __init__(
+        self,
+        input_channels: int,
+        squeeze_channels: int,
+        activation: Optional[Callable[..., nn.Module]] = None,
+    ) -> None:
         super().__init__()
         self.fc1 = nn.Conv2d(input_channels, squeeze_channels, 1)
         self.fc2 = nn.Conv2d(squeeze_channels, input_channels, 1)
+        if activation is None:
+            activation = nn.SiLU
```

Comment:
Given the original Squeeze-and-Excitation [paper](https://arxiv.org/pdf/1709.01507.pdf) uses ReLU, perhaps that's a better default option for when we move it out to the `ops` namespace?